frameworks/Java/proteus/pom.xml

```diff
@@ -214,54 +200,51 @@
 			<version>4.12</version>
 			<scope>test</scope>
 		</dependency>
-
 		<dependency>
 			<groupId>io.sinistral</groupId>
 			<artifactId>proteus-core</artifactId>
 			<version>0.1.7.4-SNAPSHOT</version>
-			
 		</dependency>
-			<dependency>
+		<dependency>
 			<groupId>com.fizzed</groupId>
 			<artifactId>rocker-runtime</artifactId>
-			<version>0.18.0</version>
+			<version>${rocker.version}</version>
 		</dependency>
 		<dependency>
 			<groupId>com.fizzed</groupId>
 			<artifactId>rocker-compiler</artifactId>
-			<version>0.18.0</version>
+			<version>${rocker.version}</version>
 			<scope>provided</scope>
 		</dependency>
-		 <dependency>
-      <groupId>com.github.spullara.mustache.java</groupId>
-      <artifactId>compiler</artifactId>
-      <version>0.9.4</version>
-    </dependency>
-
+		<dependency>
+			<groupId>com.github.spullara.mustache.java</groupId>
+			<artifactId>compiler</artifactId>
+			<version>0.9.4</version>
+		</dependency>
 		<dependency>
 			<groupId>com.zaxxer</groupId>
 			<artifactId>HikariCP</artifactId>
-			<version>2.6.1</version>
+			<version>3.2.0</version>
 		</dependency>
 		<dependency>
 			<groupId>mysql</groupId>
 			<artifactId>mysql-connector-java</artifactId>
 			<version>5.1.47</version>
 		</dependency>
-	  	<dependency>
+		<dependency>
 			<groupId>org.postgresql</groupId>
 			<artifactId>postgresql</artifactId>
-			<version>42.2.5</version>
-		</dependency>  
+			<version>9.4.1212</version>
```

Comment:
@noboomu Hi, is there a particular reason for downgrading the PostgreSQL JDBC driver?
Some time ago I've made a PR to unify the DB driver versions.

If you have found performance regressions you could try to report them. Some time ago the maintainers implemented basic test suite for this. The project lives here: https://github.com/pgjdbc/benchmarks

Reply:
@zloster I am seeing a 25% performance improvement, but I will investigate with the benchmark you linked. Thanks!

Reply:
BTW I deleted my fork by accident... should I close this PR and open another?

Reply:
Thanks for the response. A while ago I've played with the 9.4.1212 vs 42.0.0. My findings back then are here: https://www.postgresql.org/message-id/d5d39fffac03727934f262c793899321@edno.moe